pyomo/contrib/doe/doe.py

```diff
@@ -1802,6 +1815,350 @@ def compute_FIM_full_factorial(
 
         return self.fim_factorial_results
 
+    def compute_FIM_factorial(
+        self,
+        model=None,
+        design_vals: dict = None,
+        abs_change: list = None,
+        rel_change: list = None,
+        n_designs: int = 5,
+        method="sequential",
+        df_settings=(True, None, None, 500),
+        initialization_scheme="snake_traversal",
+        file_name: str = None,
+    ):
+        """Will run a simulation-based factorial exploration of the experimental input
+        space (i.e., a ``grid search`` or ``parameter sweep``) to understand how the
+        FIM metrics change as a function of the experimental design space. This method
+        can be used for both full factorial and fractional factorial designs.
+
+        Parameters
+        ----------
+        model : DoE model, optional
+            The model to perform the full factorial exploration on. Default: None
+        design_vals : dict,
```

Comment:
After reading this doc string, I do not fully appreciate all of options. I do not think we are looking at all of the edge cases with the tests.